cranelift/filetests/filetests/isa/x86/simd-bitwise-run.clif

```diff
@@ -145,6 +145,15 @@ block0:
 }
 ; run
 
+function %sshr_i64x2(i64x2, i32) -> i64x2 {
+block0(v0:i64x2, v1:i32):
+    v2 = sshr v0, v1
+    return v2
+}
+; run: %ishl_i64x2([1 -1], 0) == [1 -1]
```

Comment:
I am confused by the use of `ishl_64x2` at this point.  Shouldn't that be `ishr_64x2`?  The actual results shown are consistent with a signed shift right, not a shift left.

Reply:
Also I'd be happier if there were a couple more tests here.  Could you add a test that `1000-(63)-000 >>signed 63` produces `111-(64)-111` and that `0111-(63)-111 >>signed 63` produces `000-(64)-000` ?

Reply:
Oh, good catch; yeah, that's likely an oversight from copying other tests. I'll fix these and add the additional tests.